src/components/i18n/DopplerIntlProvider.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { IntlProvider } from "react-intl";
+import { messages_en } from "./en";
+import { messages_es } from "./es";
+import { flattenMessages } from "./utils";
+
+const messages = {
+  es: messages_es,
+  en: messages_en,
+};
+
+interface DopplerIntlProviderProps {
+  locale: string;
+  children: React.ReactNode;
+}
+
+export const DopplerIntlProvider = (props: DopplerIntlProviderProps) => {
+  return (
+    <IntlProvider
+      locale={props.locale}
+      defaultLocale="es"
+      messages={flattenMessages(
+        messages[props.locale as keyof typeof messages]
+      )}
```

Comment:
Tiene sentido @maurocardinali @andresmoschini esto y que `./dictionary `devuelva un objeto con los idiomas?
```suggestion
import { dictionary } from "./dictionary";
import { flattenMessages } from "./utils";

interface DopplerIntlProviderProps {
  locale: string;
  children: React.ReactNode;
}

export const DopplerIntlProvider = (props: DopplerIntlProviderProps) => {
  return (
    <IntlProvider
      locale={props.locale}
      defaultLocale="es"
      messages={flattenMessages(
        dictionary[props.locale as keyof typeof dictionary]
      )}
```

Reply:
No entendí

Reply:
Osea, entiendo que proponés mover esto:

```ts
import { messages_en } from "./en";
import { messages_es } from "./es";
const messages = {
  es: messages_es,
  en: messages_en,
};
```

a un módulo `dictionary`, para que?

Reply:
En ves de construir el objeto _message_ crear un archivo _dictionary_ que devuelva el objeto armado {en,es} y acá en `DopplerIntlProvider.tsx` que solo lo importe y lo use.

Reply:
No se si agrega valor ya que no es obvio lo que significa ese módulo `dictionary`, tal vez si el nombre fuera más claro...

Reply:
Quizá `translationsDictionary`